cmd/cerbosctl/hub/epdp/list_candidates.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2021-2024 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package epdp
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/alecthomas/kong"
+
+	"github.com/cerbos/cerbos/cmd/cerbosctl/internal/printer"
+	"github.com/cerbos/cerbos/internal/namer"
+	"github.com/cerbos/cerbos/internal/policy"
+	"github.com/cerbos/cerbos/internal/storage/index"
+)
+
+const listCandidatesCmdHelp = `# List candidates
+
+cerbosctl hub epdp list-candidates ./path/to/repo
+
+cerbosctl hub epdp lc ./path/to/repo
+
+# List candidates, print no headers
+
+cerbosctl hub epdp list-candidates ./path/to/repo --no-headers
+
+cerbosctl hub epdp lc ./path/to/repo --no-headers`
+
+const (
+	embeddedPDPKey       = "hub.cerbos.cloud/embedded-pdp"
+	policyNotFoundInRepo = ""
+)
+
+type loadPolicyFn func(ctx context.Context, name string) (*policy.Wrapper, error)
+
+type ListCandidatesCmd struct {
+	Path      string `arg:"" type:"path" help:"Path to repository"`
+	NoHeaders bool   `help:"Do not output headers"`
+}
+
+func (c *ListCandidatesCmd) Run(k *kong.Kong) error {
+	ctx := context.Background()
+	idx, err := index.Build(ctx, os.DirFS(c.Path))
+	if err != nil {
+		return fmt.Errorf("failed to build index: %w", err)
+	}
+
+	policyIDs, err := idx.ListPolicyIDs(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to list policies: %w", err)
+	}
+
+	tw := printer.NewTableWriter(k.Stdout)
+	if !c.NoHeaders {
+		tw.SetHeader([]string{"POLICY ID", "PATH"})
+	}
+
+	candidates, err := listCandidates(ctx, loadPolicy(idx), policyIDs...)
+	if err != nil {
+		return fmt.Errorf("failed to list candidates: %w", err)
+	}
+
+	for policyKey, policyID := range candidates {
+		tw.Append([]string{namer.PolicyKeyFromFQN(policyKey), policyID})
```

Comment:
We're already getting the policy keys here so this is unnecessary. 
```suggestion
		tw.Append([]string{policyKey, policyID})
```